man/runc-restore.8.md

```diff
@@ -74,6 +74,14 @@ daemon. See [criu --lazy-pages option](https://criu.org/CLI/opt/--lazy-pages).
 : Specify an LSM profile to be used during restore. Here _type_ can either be
 **apparamor** or **selinux**, and _label_ is a valid LSM label. For example,
 **--lsm-profile "selinux:system_u:system_r:container_t:s0:c82,c137"**.
+By default, the checkpointed LSM profile is used upon restore.
+
+**--lsm-mount-context** _context_
```

Comment:
I hate typing extra letters, why not just --mount-context?  Or --mount-label, since context is not a common user facing term.

Reply:
As a matter of fact, I think all through runc, it is referred to as the MountLabel.

Reply:
@adrianreber WDYT about `--mount-context` or `--mount-label`?

Reply:
The only reason not to rename it is to keep the options consistent between criu and runc.

Another reason not to rename it is that Podman already uses this option if it finds support for it in runc.

Reply:
OK I'll leave this decision to @rhatdan 🦆

Reply:
Yeah I'm with @kolyshkin. I think it would be nice to use something a bit less cryptic -- the fact that podman already expects this (presumably because crun already supports this?) is a bit of a shame. Maybe we can have a hidden option called `--lsm-mount-context` in order to allow podman to use it but users are pushed towards using the more friendly `--mount-label` (or `--selinux-label` as @thaJeztah suggested)?

Reply:
It can't be `--selinux-label` as the format is _type:label_, and the _type_ can currently be apparmor or selinux.

Reply:
> I hate typing extra letters, why not just --mount-context? Or --mount-label, since context is not a common user facing term.

@rhatdan I think this option is not likely to be used very often directly by users from command-line and using an explicit name clarifies the meaning of "context".

Reply:
> Yeah I'm with @kolyshkin. I think it would be nice to use something a bit less cryptic -- the fact that podman already expects this (presumably because crun already supports this?) is a bit of a shame.

I thought a bit more about the Podman problem and it is not really a problem. Podman currently checks for the existence of `--lsm-mount-context` and tells the user early if runc does not support it. So we can still call it whatever we want in runc and adapt Podman.

Should I change it to `--mount-label`? Would that be a good name for everyone?

Reply:
@kolyshkin Are you sure? The example given in the docs is `--lsm-mount-context "system_u:object_r:container_file_t:s0:c82,c137"`. `--lsm-profile` takes `type:profile`.

However, since we already have `--lsm-profile` maybe `--lsm-mount-context` is more consistent. Since it looks like we've copied the CRIU argument names in the past we should probably just stick with it...

Reply:
@cyphar right, I mixed it with `--lsm-profile` which does have `apparmor` or `selinux` prefix.

Reply:
> Since it looks like we've copied the CRIU argument names in the past we should probably just stick with it...

Fair point. Adding `--lsm-mount-label` to the list of possible names (naming is hard!)